README.md

```diff
@@ -66,16 +66,19 @@ Follow the steps below to get started with source{d} Engine.
 Follow these instructions:
 
 - [Docker for Mac](https://store.docker.com/editions/community/docker-ce-desktop-mac)
+  ```bash
+  brew cask install docker
```

Comment:
up to you, but jfyi - this is _exactly_ the same installation method - `brew cask` is just a CLI automation instead of drag-n-drop, to unpack the `.dmg` image

Reply:
The thing is, it might be obvious to most Mac users, but for the other systems we are simply copying the relevant CLI instructions from the linked documentation; and for Mac we are suggesting an alternative installation mode. Some people might understand it as you need to follow both installations as sequential steps.

Maybe it could be solved with:
- [Docker for Mac](https://store.docker.com/editions/community/docker-ce-desktop-mac). Or, if you prefer to use [Homebrew](https://brew.sh/):

Reply:
makes sense, thank you for suggesting alternative!
addressed in https://brew.sh/